### DISCUSSION TITLE
Shop Pack Groups: allow for a pack to be in multiple groups

### DISCUSSION THREAD

**Discussion by goosenation** [gameserverapp/Platform#981] [Suggestions]
Awesome addition!

# Is it possible to make it so that a pack can be in multiple groups?

As an example:
Red Gems are in a group "Resources", and also in a group "Aberration".

**Comment by mvaessen** [comment]
thanks! can you please move the second item to a separate suggestion?

**Comment by goosenation** [comment]
Done 👍✅

#985

**Reply by mvaessen** [reply]
thanks!
